protobufs/funding/funding.proto

```diff
@@ -120,3 +129,15 @@ message FundingData {
   string nft_address = 11;
   string payment_details_id = 12;
 }
+
+message FundingResponseData {
+  FundingData funding = 1;
+  repeated FundingSignature signatures = 2;
+}
+
+message FundingSignature {
+  bool valid = 1;
+  bool outdated_signature = 2;
```

Comment:
what is an outdated signature here ?

Reply:
let's say:

I have a funding with
 ```json
funding =  {"apr": 0.5}
```
If I would add a signature and afterwards change the apr value. 
`outdatedSignature=true`

Reply:
otherwise, it is `false`. It is basically tagging outdated signatures

Reply:
Do we still need to keep this `signature_outdated` field? Vs validating that if a an attribute is modified we always regenerate the signature